navigator/backend/src/main/scala/com/digitalasset/navigator/UIBackend.scala

```diff
@@ -276,7 +277,7 @@ abstract class UIBackend extends LazyLogging with ApplicationInfoJsonSupport {
 
     implicit val system: ActorSystem = ActorSystem("da-ui-backend")
 
-    val (graphQL, info, store, getAppState, partyRefresh) = setup(arguments, config)
+    val (graphQL, info, store @ _, getAppState, partyRefresh) = setup(arguments, config)
```

Comment:
I've seen this `binding @ _` pattern a few times but I'm not sure I understand this. Why not just `_`?

Reply:
readibility. This way you know which parameter is ignored.